connect/runtime/src/main/java/org/apache/kafka/connect/storage/KafkaConfigStorage.java

```diff
@@ -216,19 +215,19 @@ public KafkaConfigStorage(Converter converter, Callback<String> connectorConfigC
         offset = -1;
     }
 
-    public void configure(Map<String, ?> configs) {
-        if (configs.get(CONFIG_TOPIC_CONFIG) == null)
+    public void configure(DistributedConfig config) {
+        topic = config.getString(DistributedConfig.CONFIG_TOPIC_CONFIG);
+        if (topic == null)
```

Comment:
Is this even something that needs to be checked anymore since there is no default for this config? I suppose we could check for empty string instead.